src/components/article-list.js

```diff
@@ -45,5 +45,28 @@ export class ArticleList extends Component {
 }
 
 export default connect((state) => ({
-  articles: state.articles
+  articles: (() => {
```

Comment:
А зачем тут самовызывающаяся функция?

Reply:
```suggestion
static propTypes = {
    articles: PropTypes.array.isRequired,
    ...
}
```

потому что в пропсах у нас должен быть уже готовый отфильтрованный массив.